lib/puppetscript.rb

```diff
@@ -69,9 +69,9 @@ def self.generate_user_array(users)
     users = users.map { |user| "\"#{user.email.split('@').first}\"" }
     users.join(', ')
   end
-
+  
   def self.node_file_correct?(vm_name, contents)
-    result =    contents.lines[0].chomp.eql?('class node_' + vm_name + ' {')
+    result =    contents.lines[0].chomp.eql?('class node-' + vm_name + ' {')
```

Comment:
Same as above. If this method returns false, then the Unsupported Format is raised